lib/rig_inbound_gateway/api_proxy/handler/http.ex

```diff
@@ -116,7 +116,10 @@ defmodule RigInboundGateway.ApiProxy.Handler.Http do
         |> with_cors()
         |> Tracing.Plug.put_resp_header(Tracing.context())
         |> Conn.put_resp_content_type("application/json")
-        |> Conn.send_resp(:ok, response)
+        |> Map.update!(:resp_headers, fn existing_headers ->
+          existing_headers ++ Map.to_list(response_headers)
+        end)
```

Comment:
Doesn't existing_headers likely already include a Content-Type header?

I've looked into the HTTP grammar (1.0 and 1.1) and couldn't find anything on how often particular header keys may occur and how to deal with multiple occurrences. Therefore, I assume multiple content-type headers lead to unexpected behavior.

Reply:
This was originally supported by structured mode, but as we are dropping it, I'll close this one.